src/redux/actions/employer.ts

```diff
@@ -20,7 +20,9 @@ export const registerEmployerRequest = (
   }
 }
 
-export const registerEmployerSuccess = (employerInfo: Credential) => {
+export const registerEmployerSuccess = (
+  employerInfo: CredentialStateEmployer
```

Comment:
Why change the type here, I think `Credential` type was correct

Reply:
Oh, yeah. But I've changed the registerEmployer recently in the update-employer branch anyway. :)